### PR TITLE
Add support for GitHub Actions

### PR DESCRIPTION
**Pull request by nobreconfrade**
I only add build for 3.7 python on linux and windows latest. But if others are necessary just tell me. Also where those 'Publish package to PyPI' variables are set? If someone could tell me just send a PM.

### CODE REVIEW

**Comment by nicoddemus** [comment]
Please run linting locally to fix it. 👍

**Comment by nobreconfrade** [comment]
> Please run linting locally to fix it. +1

Trying to but when I run it i receive an error: 
`ModuleNotFoundError: No module named 'pre_commit_hooks'`.  Need to check why pre commit isn't found.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+name: build
+
+on: [push, pull_request]
```
Comment (williamjamir) [inline .github/workflows/main.yml]:
```suggestion
on:
  push:
    branches:
      - master
      - rb-*

  pull_request:
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
```
Comment (williamjamir) [inline .github/workflows/main.yml]: Also adds a schedule
```suggestion
schedule:
  - cron: 0 2 * * *
```

Comment (nobreconfrade) [reply .github/workflows/main.yml]: Done

**Comment by nicoddemus** [comment]
Awesome, thanks!

**Pull request merged**